analyzers/tests/SonarAnalyzer.UnitTest/Rules/BreakOutsideSwitchTest.cs

```diff
@@ -25,8 +25,16 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class BreakOutsideSwitchTest
     {
+        private readonly VerifierBuilder verifier = new VerifierBuilder<BreakOutsideSwitch>();
+
         [TestMethod]
         public void BreakOutsideSwitch() =>
-            new VerifierBuilder<BreakOutsideSwitch>().AddPaths("BreakOutsideSwitch.cs").Verify();
+            verifier.AddPaths("BreakOutsideSwitch.cs").Verify();
+
+        [TestMethod]
+        public void BreakOutsideSwitch_CSharp7() =>
```

Comment:
There is no need to create a C# 7 test because it's the default min language version if you pass no `ParseOptions`.